charts/centrifugo/templates/ingress.yaml

```diff
@@ -29,9 +29,12 @@ spec:
         paths:
           {{- range .paths }}
           - path: {{ . }}
+            pathType: ImplementationSpecific
```

Comment:
I'd like to change this file to use the `defaultBackend` like we do for the Kami library:

https://github.com/KamiHQ/kami-library/blob/main/deploy/gke/production/ingress.yml

Without this you get an extra backend created on the Google load balancer, but it's not used as far as I can tell.

I can do this as a separate change though.